hudi-client/src/test/java/org/apache/hudi/TestHoodieClientOnCopyOnWriteStorage.java

```diff
@@ -79,9 +82,22 @@
 import static org.mockito.Mockito.when;
 
 @SuppressWarnings("unchecked")
+@RunWith(Parameterized.class)
 public class TestHoodieClientOnCopyOnWriteStorage extends TestHoodieClientBase {
 
   private static final Logger LOG = LogManager.getLogger(TestHoodieClientOnCopyOnWriteStorage.class);
+  private final IndexType indexType;
+
+  @Parameterized.Parameters(name = "{index}: Test with IndexType={0}")
+  public static Collection<Object[]> data() {
+    Object[][] data =
+        new Object[][] {{IndexType.BLOOM},{IndexType.GLOBAL_BLOOM},{IndexType.SIMPLE}};
```

Comment:
@vinothchandar @bvaradar : As part of testing the new simple index, I have parametrized this test (TestHoodieClientOnCopyOnWriteStorage) and have added other index types as well. is that fine or do you think the increase in build/test time is not worth it.

Reply:
how much does it increase by... doing the full suite with both indexes may be an overkill IMO

Reply:
aprrox 2 mins per index type.

Reply:
Could be okay then